### PR TITLE
my first commit in this file1

### PR DESCRIPTION
**Pull request by lacaine**

### CODE REVIEW

**Comment by lacaine** [comment]
please kindly rew

**Comment by lacaine** [comment]
please kindly review and advise

**Comment by rnfor-pro** [comment]
awesome

**Comment by rnfor-pro** [comment]
Good Lacaine

File: file1 [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ I exercise myself
```
Comment (rnfor-pro) [inline file1:1]: correct this code

**Pull request closed without merging**